### ISSUE TITLE
Get Distributed Tracing Working in Sentry

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#12230]
Sentry now supports distributed tracing. We have transactions being recorded, which can be seen in the performance tab. Investigating these events will show timings for each transaction, but outbound transactions are not being recorded. This is supported by ‘[distributed tracing.](https://docs.sentry.io/product/sentry-basics/tracing/distributed-tracing/)'

Acceptance Criteria:

* Transactions can be linked across requests
* Ideally, full set of spans required to full a request from the client can be seen in the ‘Event Details’ section in sentry.
* Outbound database calls are visible
* Outbound HTTP requests are visible
* Outbound Redis calls are visible
* Basically any service being invoked should show up in the transaction as a span.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4748)

**Comment by cknowles-admin** [comment]
FxA is no longer synchronizing all issues between Jira and Github.  We are closing open issues and will selectively synchronize in the future.
